packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -35,7 +35,33 @@ AssertExists() {
   return 0
 }
 
+# Check if the project.pbxproj file will end up embedding Flutter.framework multiple
+# times (once for Cocoapods and once from the template).  This will cause cryptic 
+# build errors in Xcode 10, and we don't have a good way to remedy it without
+# potentially damaging the project configuration.
+ValidateFlutterFrameworkXcode10() {
+  # Bail out if we're below Xcode 10
+  if [[ "Xcode 9.9.9" != "$(printf "$(xcodebuild -version | head -n1 )\nXcode 9.9.9" | sort -V | head -n1)" ]]; then
+    return 0
+  fi
+
+  PROJECT_FILE_PATH="/Users/dnfield/src/ios_test_mod/.ios/Runner.xcodeproj"
```

Comment:
`/Users/dnfield` is probably not what was intended here.

Reply:
Ahh this was for testing.  Thanks!